libs/ledger-live-common/src/families/cosmos/chain/cosmosBase.ts

```diff
@@ -9,7 +9,8 @@ abstract class cosmosBase {
   abstract prefix: string;
   defaultGas = 100000;
   minGasPrice = 0.0025;
-  minimalTransactionAmount = new BigNumber(10000);
+  // minimalTransactionAmount is only used in bot tests. When we make a transaction ,we should make sure that the spendable balance is greater than minimalTransactionAmount. We usually use the upper limit of send transaction fee as the minimalTransactionAmount. e.g. 5000 uatom for cosmos.
```

Comment:
Shouldn't this be an input of "generateGenericCosmosTest" method instead ? We don't want test logic in our app imo

In specs.ts we do :
 const cosmos = {
  ...generateGenericCosmosTest("cosmos"),
};

Reply:
fixed